packages/frontend/src/Components/Search.tsx

```diff
@@ -27,14 +37,16 @@ const Search = () => {
         variant="filled"
         placeholder="Search for a song"
         onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-          // TODO: Phase 2: Update searchText state
+          setSearchText(e.target.value)
+          console.log(e.target.value)
```

Comment:
Super minor but remove any log statements that you might have added! It's more of a performance thing